pkgs/tools/admin/coldsnap/default.nix

```
@@ -0,0 +1,30 @@
+{ openssl
+, pkg-config
+, rustPlatform
+, fetchFromGitHub
+, lib
```

It's better to keep this convention as it's used pretty much everywhere in Nixpkgs.
```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
, openssl
, pkg-config
```
You should also add the `Security` input so that this package can be built on Darwin, currently it [does not](https://github.com/NixOS/nixpkgs/pull/196035/checks?check_run_id=8938765063).